### ISSUE TITLE
not compatible with some gulp plugins with Bluebird.promisifyAll(fs)

### ISSUE THREAD

**Issue by zeratul1** [giakki/uncss#341]
Hi there,

I found a problem when build [essential-image-optimization](https://github.com/GoogleChrome/essential-image-optimization) 

**uncss** execute pormisifyAll with `{ multiArgs: true }`
https://github.com/giakki/uncss/blob/a0a6fbd869dcca256506533a4d9f8abc5b7c3700/src/utility.js#L10

while other packages would prefer the default behavior
```
var fs = promise.promisifyAll(require('fs')), 
```

> Setting `multiArgs` to `true` means the resulting promise will always fulfill with an array of the callback's success value(s). 

So any modules calling `fs.readFileAsync` and expect the result of a Stream or a Buffer will break.

In my case, the critical module.
```
return fs.readFileAsync(file.path).then(contents => {
    file.contents = contents;
    return file;
});
```

https://github.com/addyosmani/critical/blob/fc23eb19fd29b51a8449767275e2d4295d5b4692/lib/file-helper.js#L347

Could `uncss` change its `promisifyAll(require('fs'))` to the default options?

**Comment by RyanZim** [comment]
@zeratul1 There is a work-in-progress branch that migrates away from Bluebird to native promises. That should fix this issue. This branch's changes will be released in the next major release.

**Comment by zeratul1** [comment]
Cool, 
that is great :D

Change to native promises should fix my issue.

**Comment by RyanZim** [comment]
This has been fixed since 0.16.0; closing.
